README.md

```diff
@@ -33,11 +33,21 @@ You can download any icon of the pack directly from https://iconoir.com or get t
 ## Font
 
 Iconoir is available in .EOT, .SVG, .TTF, and .WOFF. Get them in the /fonts folder.
+If you want to use Iconoir in a HTML file, you just need to add the [iconoir.css](./fonts/iconoir.css) file.
+
+Import the CSS File:
+
+```html
+<link rel="stylesheet" href="https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.css">
```

Comment:
This overflows on smaller screens, use relative URL instead?

`<link rel="stylesheet" href="./iconoir/fonts/iconoir.css">`

Reply:
Or put into two lines:

```
<link rel="stylesheet"
      href="https://raw.githubusercontent.com/lucaburgio/iconoir/master/fonts/iconoir.css">
```

Reply:
The long version of the link is so that the CSS file does not have to be downloaded.

Reply:
And on smaller devices you can scroll in the code line. Writing the code in 2 lines makes no difference and doesn't look nice.